src/javascript/app_2/Constants/routes.js

```diff
@@ -0,0 +1,19 @@
+const routes = {
+    account_password: '/account_password',
+    apps            : '/apps',
+    cashier_password: '/cashier_password',
+    exclusion       : '/exclusion',
+    financial       : '/financial',
+    history         : '/history',
+    index           : '/index',
+    limits          : '/limits',
+    personal        : '/personal',
```

Comment:
Is it possible to be more explicit here?
i.e. `/personal` finally would become `/settings/personal`, but the way stated here is unclear and confusing.

Reply:
@binary-hamedanchi 
it's possible.